### PR TITLE
Jerry development

### PR DESCRIPTION
**Pull request by jerryli0273**
Completed the portfolio page.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,284 @@
...
+                <a href="projects.html" class="btn btn-outline-secondary">More details</a>
+            </div>
+        </div>
+    </div>
+    <div class="col-md-6">
+        <div class="card">
+            <img class="card-img-top" src="images/wildfire.jpg">
+            <div class="card-body">
+                <h4 class="card-title">UCB Research</h4>
+                <p class="card-text">A project invovling using small devices
```
Comment (mk-valentine) [inline portfolio/src/main/webapp/index.html:172]: such a cool project!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,284 @@
...
+                    satellite images.</p>
+                <a href="projects.html" class="btn btn-outline-secondary">More details</a>
+            </div>
+        </div>
+    </div>
+</div>
+</div>
+
+
+<!--- Hobbies header -->
```
Comment (mk-valentine) [inline portfolio/src/main/webapp/index.html:183]: Are you using this and the portfolio/src/main/webapp/hobbies.html? Maybe remove whichever one you aren't using?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,284 @@
...
+    </div>
+</div>
+</div>
+
+<button onclick="getFetch()">Fetch!</button>
+<div class="col-lg-6" id="fetch-container"></div>
+
+<form action="/data" method="POST">
+
+      <p>Enter your name and your comment separated by a comma:</p>
```
Comment (mk-valentine) [inline portfolio/src/main/webapp/index.html:242]: what happens if there are other commas in the comment?

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -26,3 +27,26 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches the information on the /data page and presents it in the fetch
```
Comment (mk-valentine) [inline portfolio/src/main/webapp/script.js:32]: Nice job with the function comments! Makes your code easy to read :)

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (jerryli0273) [reply portfolio/src/main/webapp/index.html:191]: deleted!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (jerryli0273) [reply portfolio/src/main/webapp/index.html]: restructured the comments input structure

**Pull request merged**